validator_manager/src/import_validators.rs

```diff
@@ -80,14 +83,94 @@ pub fn cli_app() -> Command {
                 )
                 .display_order(0),
         )
+        .arg(
+            Arg::new(PASSWORD)
```

Comment:
Is the `--password` flag only valid when using `--standard-format`? If that's the case, we should add a `.requires(STANDARD_FORMAT)` to its cli definition.  This is kind of circular but I think it should be fine. Without this, a user would be able to use the `--password` flag when using the Lighthouse format which would be silently ignored.

Reply:
Added in [7d9346a](https://github.com/sigp/lighthouse/pull/6261/commits/7d9346ab2d28d2dc1d999e3e4a905a44105493aa)